src/App.jsx

```diff
@@ -14,57 +16,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodSelected, setGoodSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {goodSelected === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {goodSelected} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setGoodSelected(null)}
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map(item => {
+            return (
+              <tr
+                data-cy="Good"
+                key={item}
```

Comment:
You are using the 'item' as a key for each element in the map. This is not recommended, as per the second point of the checklist: 'DON'T use index as a key'. If the 'item' values are unique, it's fine to use them as keys, but if they are not unique, this can cause issues with React's rendering. If the 'goods' array can have duplicate values, consider using a unique identifier for each good instead.